pkg/cluster/clusterfiles.go

```diff
@@ -36,7 +36,7 @@ Description=rktlet: The rkt implementation of a Kubernetes Container Runtime
 Documentation=https://github.com/kubernetes-incubator/rktlet/tree/master/docs
 
 [Service]
-ExecStart=/usr/bin/rktlet --net=weave
+ExecStart=/usr/bin/rktlet --net=%s
```

Comment:
I think you probably wanted to say `--net={{.CNIModel}}`.
Though I'm not sure if all CNI models work fine with rktlet by doing this. I'll check later.